.travis.yml

```diff
@@ -0,0 +1,104 @@
+
+language: generic
+sudo: required
+dist: trusty
+
+matrix:
+  fast_finish: true
+
+  include:
+    - os: linux
+      addons:
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-6', 'cmake', 'python3', 'python3-pip', 
+'python3-setuptools', 'nodejs', 'npm', 'libsfml-dev', 'libprotobuf-dev', 
+'protobuf-compiler', 'libzmq-dev']
```

Comment:
Here and below: we will only need the packages g++-6 (and later on cmake) from apt.